master/internal/api_master.go

```diff
@@ -110,17 +109,11 @@ func (a *apiServer) PatchMasterConfig(
 	for _, path := range paths {
 		switch path {
 		case "log":
-			if !isValidLogLevel(req.Config.Log.Level) {
-				panic(fmt.Sprintf("invalid log level: %s", req.Config.Log.Level))
-			}
-			a.m.config.Log.Level = req.Config.Log.Level
+			a.m.config.Log.Level = req.Config.Log.Level.String()[10:]
```

Comment:
what is this `[10:]`? is it trimming `LOG_LEVEL`...? If so, we should just write a `ProtoToLogrusLevel` beside `LogrusLevelToProto` in `pkg/logger`, since this would break in strange ways if someone renamed / moved that enum.